src/statmodels.jl

```diff
@@ -423,6 +423,29 @@ Return all parameters of a model.
 params(model) = error("params is not defined for $(typeof(model))")
 function params! end
 
+"""
+    vif(model, param)
+
+Return the variance inflation factor for a parameter in a model.
```

Comment:
How about saying "coefficient" rather than "parameter"? That's how we call these in general (function `coef` and type `CoefTable`, while `params` returns distributions parameters IIRC).

Reply:
:+1: Yeah, `coef` is also a better match if we restrict to `RegressionModel`.